README.md

```diff
@@ -1,105 +1,3 @@
 # Regulatory R Package Repository
 
-Using R in submissions to healthcare regulators is challenging since the 
-quality of any packages used must be ensured and evidence of ensuring this
-quality must be made available on request. 
-This working group strives to identify and prototype at least one technical
-framework that can support a transparent, open, dynamic, cross-industry approach
-of establishing and maintaining a 'repository' of R packages with
-accompanying evidence of their quality and the assessment criteria, that can be used to simplify necessary
-in-house validation processes as much as possible.
-Such a cross-industry 'repository' could help with burden-sharing of
-validation efforts, improve quality via transparent, open peer review, and
-de-risk the use of public R packages for regulatory submissions.
-We seek a pragmatic solution, ideally endorse by key regulators.
-Even though we strive to keep the approach as open as possible the initial
-focus lies on the pharmaceutical industriy and their regulatory environment. 
-
-This document details the scope and initial short-term goals supporting the
-creation of a cross-pharma regulatory package repository.
-
-## How to get involved
-
-- Reply to our ["sign up" thread](https://github.com/pharmaR/regulatory-r-repo-wg/issues/1)
-  to be contacted about planning meetings.
-- Sign up for the [R Validation Hub mailing list](https://www.pharmar.org/contact/)
```

Comment:
Maybe we can keep some of this in the README in case people discover the repo before the webpage. 

I think having the vision statement, "how to get involved" and a link to the webpage to learn more would be sufficient, we can still cut most of it.

Reply:
You are right, I have re-added these paragraphs.